composer.json

```diff
@@ -23,9 +23,9 @@
         "sensio/framework-extra-bundle"        : "^3.0",
         "symfony/monolog-bundle"               : "^2.8",
         "symfony/swiftmailer-bundle"           : "^2.3",
-        "symfony/symfony"                      : "^3.2-beta1",
+        "symfony/symfony"                      : "^3.2-rc1",
         "twig/extensions"                      : "^1.3",
-        "twig/twig"                            : "^1.27",
```

Comment:
Yes and no. As you can see in [the CHANGELOG of 3.2-rc1](https://github.com/symfony/symfony/pull/20552): 

```
bug #20484 bumped min version of Twig to 1.28 (@fabpot)
```

When some minimum version (e.g. `1.28`) is mandatory for some dependency (e.g. `Twig`) I like to set it explicitly.